core/node/main/config.cpp

```diff
@@ -166,7 +166,7 @@ namespace fc::node {
                                                std::end(drand_flags),
                                                [](bool x) { return x; })};
     if (drand_flags_count == 0) {
-      config.drand_servers.push_back("api.drand.sh");
+      config.drand_servers.push_back("api2.drand.sh");
```

Comment:
Maybe you could want to change url in `core/drand/example/drand_example.cpp`?

Reply:
Discussed. It is better to have config for such parameters, but it is out of PR scope.